doc/neo-tree.txt

```diff
@@ -148,6 +148,38 @@ commands by adding this option before the Neotree config is run:
 >vim
   let g:neo_tree_remove_legacy_commands = 1
 <
+CALLING_FROM_LUA
+
+You can call Neotree commands from your Lua scripts as follows:
+
+>lua
+  require('neo-tree.command').execute({ ... })
+
+For example, in the `keys` configuration, you could use the following to make
+the `-` key reveal the current file, or if in an unsaved file, the current
+working directory.
+
+>lua
+
+  keys = {
```

Comment:
What is `keys`? It doesn't have anything to do with Neo-tree and I don't think this code on its own would do anything. I could be missing out on something though.